packages/vc-verification/src/issuer-verification-vc.ts

```diff
@@ -8,7 +8,7 @@ import { verifyCredential } from 'didkit-wasm-node';
  * The hierachy must only consist of VC issuance
  */
 export class VCIssuerVerification {
-  private _issuerDefResolver: IssuerDefinitionResolver;
+  private _issuerDefResolver: AuthorityResolver;
```

Comment:
Given above integration segregation comment, the required interface can be narrowed, I believe
```suggestion
  private _issuerDefResolver: IssuerResolver;
```